docker-compose.circle.yml

```diff
@@ -22,6 +22,25 @@ xenial:
     - xenialbuild
     - xenialtest
 
+bionic:
+  image: quay.io/stackstorm/packagingrunner
+  extends:
+    file: docker-compose.override.yml
+    service: suite-compose
+  environment:
+    - BUILDNODE=bionicbuild
+    - TESTNODE=bionictest
+    # NOTE: We don't build st2mistral package so we don't mix Python 3
+    # (StackStorm components) and Python 2 (Mistral)
+    # Bionic will be first release without a Mistral
+    - ST2_PACKAGES=st2
```

Comment:
Try to set an `ST2_PACKAGES` ENV in docker-compose definition for both `bionicbuild` and `bionictest` (instead of `buildenv_st2.sh`)

Reply:
Thanks - I was actually thinking about trying that, but then I checked at the code and assumed buildenv stuff still has priority based on the way how it's passed to docker compose.

Reply:
It looks like it indeed doesn't work - https://circleci.com/gh/StackStorm/st2-packages/3180#queue-placeholder/containers/2 and I will need to go with my original approach.

As I mentioned above, I wanted to try this first then I checked docker compose script and I assumed it won't work because of the way strict is structured.

Reply:
And so we use the one taking most precedence via `-e` :/